src/confcom/azext_confcom/tests/latest/test_confcom_fragment.py

```diff
@@ -465,6 +476,173 @@ def test_fragment_injected_sidecar_container_msi(self):
         self.assertEqual(image._workingDir, expected_workingdir)
 
 
+class FragmentPolicyGeneratingTarfile(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        path = os.path.dirname(__file__)
+        cls.path = path
+
+    def test_tar_file_fragment(self):
+        custom_arm_json_default_value = """
```

Comment:
could this be changed to a fragment config and use `load_policy_from_json` instead? The code paths are slightly different, and fragments are the main culprit to the bug